<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/viewer.png</normaloff>../images/viewer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Image List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Image Viewer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>Search Image...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="image_list_widget"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="image_viewer_tab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Home</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusdsd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_image"/>
    <addaction name="action_open_images"/>
    <addaction name="action_open_folder"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menusdsd"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="tmpOpen"/>
   <addaction name="action_clear_list"/>
  </widget>
  <action name="tmpOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../pyqttest/images/open.png</normaloff>../../pyqttest/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="action_open_image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="action_open_images">
   <property name="text">
    <string>Open Images</string>
   </property>
  </action>
  <action name="action_open_folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="action_clear_list">
   <property name="icon">
    <iconset>
     <normaloff>../images/clear.png</normaloff>../images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
